### DISCUSSION TITLE
Workplace search file link problem

### DISCUSSION THREAD

**Discussion by paksak** [dadoonet/fscrawler#1521] [Q&A] [answered]
Hi,
I crawled some documents with fscrawler on Elasticsearch 8 .
It worked well and I could search document contents on Workplace   Search.
But when I try to open file link in workplace search, It doesn't work.
Link URL is "127.0.0.1/xxx" and I wonder if the problem is about path configuration.
How can I fix it?

![화면 캡처 2022-10-18 101305](https://user-images.githubusercontent.com/106945652/196312450-8ba61801-9ea6-4def-833b-565e5b485634.png)

**Comment by dadoonet** [comment, marked as answer]
Hey!

You need to run a local web server which will serve your files.
FSCrawler does not run this for you.

You can do something like this:

```sh
cd path/to/docs; python3 -m http.server --cgi 80
```

I should add this to the documentation. Would you like to contribute a PR for this?

**Comment by paksak** [comment]
Thank you so muchfor your fast answer.
I will try as you said and do PR.
Hope you have a nice day.


2022년 10월 18일 (화) 오전 11:46, David Pilato ***@***.***>님이 작성:

**Comment by paksak** [comment]
I tried as you said and solved problem.
I turned on web server and now I can open the file.
Thank you so much, David.

![화면 캡처 2022-10-18 132241](https://user-images.githubusercontent.com/106945652/196335799-266da1ee-2926-4357-8046-726edd3f0eb0.png)
![화면 캡처 2022-10-18 132404](https://user-images.githubusercontent.com/106945652/196335812-e2c0bdcf-ac9c-45d4-9b60-6c459d3d2a5b.png)
